daffodil-runtime2/src/main/resources/org/apache/daffodil/runtime2/examples/NestedUnion.c

```diff
@@ -25,7 +25,7 @@
 
 // Initialize our program's name and version
 
-const char *argp_program_version = "daffodil-runtime2 3.1.0-SNAPSHOT";
+const char *daffodil_program_version = "daffodil-runtime2 3.1.0-SNAPSHOT";
```

Comment:
Is this hardcoded version going to cause tests to fail when we update the version?

Reply:
No, this whole file is just a very lightly edited copy of an autogenerated file to be shown as an example of what generated code looks like.  We've had some discussion about failing the CI workflow if there's a difference (which would be a hassle) or auto-updating these example files in a sbt build (which would be better).  If we wanted to auto-update these files, we would have to delete the copyrights and change the filenames (unless you decide it's fine for the code generator to include the Apache copyright in its generated_code.[ch] files).